src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/overlays/frame-visualizer.ts

```diff
@@ -49,6 +49,7 @@ export class FrameVisualizer extends EntitiesVisualizer {
 			const features = this.source.getFeatures();
 			if (features && features[0]) {
 				delete (<any>features[0]).styleCache;
+				// TODO: Do we want to re-render the layer? if so, use changed().
 				this.source.refresh();
```

Comment:
what are the differences between changed() and refresh()?

Reply:
In OL6 'changed' is supposed to re-render the layer. They changed things a little bit...